app-admin/passwordsafe/passwordsafe-1.16.0.ebuild

```diff
@@ -0,0 +1,88 @@
+# Copyright 2021-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+WX_GTK_VER="3.0-gtk3"
+
+inherit cmake desktop flag-o-matic optfeature wxwidgets xdg
+
+MY_PV="${PV/_beta/BETA}"
+DESCRIPTION="Password manager with wxGTK based frontend"
+HOMEPAGE="https://pwsafe.org/ https://github.com/pwsafe/pwsafe/"
+SRC_URI="https://github.com/pwsafe/pwsafe/archive/${MY_PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/pwsafe-${MY_PV}"
+
+LICENSE="Artistic-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="qr test +xml yubikey"
+RESTRICT="!test? ( test )"
+
+DEPEND="
+	net-misc/curl
+	sys-apps/util-linux
+	x11-libs/libXt
+	x11-libs/libXtst
+	x11-libs/wxGTK:${WX_GTK_VER}[X]
+	qr? ( media-gfx/qrencode )
+	xml? ( dev-libs/xerces-c )
+	yubikey? ( sys-auth/ykpers )"
+RDEPEND="${DEPEND}"
+BDEPEND="
+	app-arch/zip
+	virtual/pkgconfig
+	sys-devel/gettext
+	test? ( dev-cpp/gtest )"
+
+pkg_pretend() {
+	einfo "Checking for -std=c++11 support in compiler"
```

Comment:
This is wrong for binary packages/installs.

Reply:
@thesamesam can you please explain which line you are talking about? I have no idea....

Reply:
On a conceptual level you don't have to, or to be precise, you **can't** perform this check when you're installing passwordsafe's binpackage. There's a way you can check whether you're installing a binpkg or compiling the product, 
`if ! [[ ${MERGE_TYPE} == binary ]]; then```or something a like. (can check `if ... == source`).

Reply:
Mmm no, src_compile will be skipped completely when using binpackages. This check should be removed. 

But there's some issue with gtest I believe, 
```
In file included from /usr/include/gtest/gtest-message.h:57,
                 from /usr/include/gtest/gtest-assertion-result.h:46,
                 from /usr/include/gtest/gtest.h:64,
                 from /var/tmp/portage/app-admin/passwordsafe-1.16.0/work/pwsafe-1.16.0/src/test/CommandsTest.cpp:19:
/usr/include/gtest/internal/gtest-port.h:270:2: error: #error C++ versions less than C++14 are not supported.
  270 | #error C++ versions less than C++14 are not supported.
      |  ^~~~~
In file included from /usr/include/gtest/gtest-printers.h:115,
                 from /usr/include/gtest/gtest-matchers.h:48,
                 from /usr/include/gtest/internal/gtest-death-test-internal.h:47,
                 from /usr/include/gtest/gtest-death-test.h:43,
                 from /usr/include/gtest/gtest.h:65:
/usr/include/gtest/internal/gtest-internal.h:636:58: error: wrong number of template arguments (0, should be 1)
```

So currently test phase is broken. 

But GUI and CLI seem to work at least I can use them.

Reply:
> Mmm no, src_compile will be skipped completely when using binpackages. This check should be removed.
> 
> But there's some issue with gtest I believe,
> 
> ```
> In file included from /usr/include/gtest/gtest-message.h:57,
>                  from /usr/include/gtest/gtest-assertion-result.h:46,
>                  from /usr/include/gtest/gtest.h:64,
>                  from /var/tmp/portage/app-admin/passwordsafe-1.16.0/work/pwsafe-1.16.0/src/test/CommandsTest.cpp:19:
> /usr/include/gtest/internal/gtest-port.h:270:2: error: #error C++ versions less than C++14 are not supported.
>   270 | #error C++ versions less than C++14 are not supported.
>       |  ^~~~~
> In file included from /usr/include/gtest/gtest-printers.h:115,
>                  from /usr/include/gtest/gtest-matchers.h:48,
>                  from /usr/include/gtest/internal/gtest-death-test-internal.h:47,
>                  from /usr/include/gtest/gtest-death-test.h:43,
>                  from /usr/include/gtest/gtest.h:65:
> /usr/include/gtest/internal/gtest-internal.h:636:58: error: wrong number of template arguments (0, should be 1)
> ```
> 
> So currently test phase is broken.
> 
> But GUI and CLI seem to work at least I can use them.


I was under the misguided impression that this PR would pull https://github.com/gentoo/gentoo/commit/226042015d009bb30121508182b19a31131d175 before merge to address this.

Reply:
Patch seems to apply cleanly to 1.16.0 and then tests work again, I'll include the patch, thanks!